src/incubator/WheelPicker/Item.tsx

```diff
@@ -35,31 +34,30 @@ export default ({
   activeColor = Colors.primary,
   inactiveColor = Colors.grey20,
   style,
-  testID
+  testID,
+  centerH = true
 }: InternalProps) => {
-  
   const selectItem = useCallback(() => onSelect(index), [index]);
   const itemOffset = index * itemHeight;
 
   const color = useMemo(() => {
     return interpolateColors(offset, {
-      inputRange: [
-        itemOffset - itemHeight,
-        itemOffset,
-        itemOffset + itemHeight
-      ],
+      inputRange: [itemOffset - itemHeight, itemOffset, itemOffset + itemHeight],
       outputColorRange: [inactiveColor, activeColor, inactiveColor]
     });
   }, [itemHeight]);
-  
+
   return (
     <AnimatedTouchableOpacity
       activeOpacity={1}
       style={{
-        height: itemHeight
+        height: itemHeight,
+        minWidth: Spacings.s10
       }}
```

Comment:
Use StyleSheet for static styles. 
And I suggest memoizing the style we pass here, It can save performance.